ci/build.sh

```diff
@@ -93,20 +101,16 @@ function push_helm_chart() {
   set -x
 
   git checkout -- .
-  sudo helm init --client-only
-  sudo helm repo add falcosecurity https://falcosecurity.github.io/charts
-  sudo helm repo add bitnami https://charts.bitnami.com/bitnami
-  sudo helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
```

Comment:
Why we don't need this here but we ned it below? 🤔  in `generate_overrides`

Reply:
we don't need them below too